ocaml/typing/printtyped.ml

```diff
@@ -305,7 +305,7 @@ and pattern : type k . _ -> _ -> k general_pattern -> unit = fun i ppf x ->
   | Tpat_record (l, _c) ->
       line i ppf "Tpat_record\n";
       list i longident_x_pattern ppf l;
-  | Tpat_array (am, l) ->
+  | Tpat_array (am, _arg_sort, l) ->
```

Comment:
Print the new field. This is used for debugging, so we don't want to hide this value.